client/src/modules/home/index.tsx

```diff
@@ -45,18 +48,24 @@ const Welcome = ({ user }: { user: User }) => {
 };
 const Home = () => {
   const [hasMore, setHasMore] = useState(true);
-  const { loading, error, data, fetchMore } = useHomeQuery({
-    variables: { offset: 0, limit: 2 },
+  const {
+    loading: chapterLoading,
+    error: chapterError,
+    data: chapterData,
+  } = useChaptersQuery();
+
+  const { loading, error, data, fetchMore } = usePaginatedEventsWithTotalQuery({
+    variables: { offset: 0, limit: eventCards },
   });
   const { user } = useUser();
 
   const toast = useToast();
   const onLoadMore = async () => {
     try {
       const res = await fetchMore({
-        variables: { offset: data?.paginatedEvents.length },
+        variables: { offset: data?.paginatedEventsWithTotal.length },
       });
-      setHasMore(res.data.paginatedEvents.length > 0);
+      setHasMore(res.data.paginatedEventsWithTotal.length > 0);
```

Comment:
I have to many possibilities, while knowing the values are always `2` and `7`. BTW, did you know, you can't mutate them, I didn't.

```ts
const total = data?.paginatedEventsWithTotal.map(({total})=> total).reduce(Number)
      setHasMore(res.data.paginatedEventsWithTotal.length < total)
      setHasMore(res.data.paginatedEventsWithTotal.length > total);
      setHasMore(res.data.paginatedEventsWithTotal.length + 2 < total)
      setHasMore(res.data.paginatedEventsWithTotal.length + 2 > total)
      setHasMore(res.data.paginatedEventsWithTotal.length += 2 < total)
      setHasMore(res.data.paginatedEventsWithTotal.length += 2 < total)
      ```